Add tryout section for Choreo Insights API (#297)
wso2/docs-choreo-dev: en/docs/insights/choreo-insights-api.md

Choreo Insights API is a GraphQL API that allows you to fetch data in a flexible manner by defining the criteria with precision. This guide explains the different ways in which you can fetch data via this API.

 - **Allowed operations:** [Queries](#query)
 - **Schema defined types:** [Objects](#objects), [Inputs](#inputs), [Enums](#enums) and [Scalars](#scalars)

##Try out

> You can try out the Choreo Insights API with your data using GraphQL Explorer from [here.](../graphiql-explorer/)

Reviewer: ```suggestion
You can try out the Choreo Insights API with your data via the [GraphQL Explorer](../graphiql-explorer/).
```

Author: Fixed with 2a243d1

!!! warning

>     GraphQL Explorer is not working in the Safari web browser due to an [issue](https://developer.apple.com/forums/thread/658688) in the browser.

Reviewer: ```suggestion
You cannot work with the GraphQL Explorer with the Safari web browser at present due to [this issue](https://developer.apple.com/forums/thread/658688) in the browser.
```

Author: Fixed with 2a243d1
